zio/logging/src/main/scala/tofu/logging/package.scala

```diff
@@ -0,0 +1,30 @@
+package tofu
+
+import org.slf4j.LoggerFactory
+import tofu.logging.Logging.loggerForService
+import tofu.logging.impl.{UIOZLogging, URIOZLoggingImpl}
+import zio.{UIO, URIO}
+
+import scala.reflect.ClassTag
+
+package object logging {
+  type ZLogs[R]    = Logs[UIO, URIO[R, *]]
+  type ZLogging[R] = Logging[URIO[R, *]]
+
+  private[logging] object ZLogsOps {
+    val uio: ZLogs[Any] = new ZLogs[Any] {
+      def forService[Svc: ClassTag]: UIO[Logging[UIO]] =
+        UIO.effectTotal(new UIOZLogging(loggerForService[Svc]))
+      def byName(name: String): UIO[Logging[UIO]] = UIO.effectTotal(new UIOZLogging(LoggerFactory.getLogger(name)))
+    }
+
+    def withContext[R: Loggable]: ZLogs[R] = new ZLogs[R] {
+      def forService[Svc: ClassTag]: UIO[ZLogging[R]] =
+        UIO.effectTotal(new URIOZLoggingImpl(loggerForService[Svc]))
+      override def byName(name: String): UIO[ZLogging[R]] =
+        UIO.effectTotal(new URIOZLoggingImpl[R](LoggerFactory.getLogger(name)))
+    }
+  }
+
+  val ZLogs: ZLogsOps.type = ZLogsOps
```

Comment:
That is unusual encoding, what does that fix?
Also why do we define `package object logging` in the module that doesn't own `package logging`
What if we decide to define `package object logging` in the logging core?
Would it be more wise define those in the `tofu.logging.zio` package?

Reply:
I just wanted to keep structure `tofu.logging.ZLogs`, but I totally agree that it will be much better to make separate `tofu.logging.zio`. Otherwise if we decide to define package object in core it will be unaccessible.
Fixed.